content/en/docs/release_notes/3_0_2_20211105_relnotes.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Axway CLI 3.0.2 GA release notes - 5 November 2021
+linkTitle: Axway CLI 3.0.2 GA release notes - 5 November 2021
+description: Review Axway CLI release details such as new features, improvements, and bug fixes.
+weight: 10
+date: 2021-11-05
+---
+
+{{% variables/axway_cli_prod_name %}} 3.0.2 GA - 5 November 2021 is a major release that includes new features, improvements, and bug fixes.
+
+## Installation
+
+```
+npm i -g axway
+```
+
+## New Features
+
+* auth: Display current region after logging in, the list of authenticated accounts, and when running `whoami`.
+
+## Improvements
+
+* auth: Validate authenticated account sessions and access tokens when listing accounts.
```

Comment:
Change Validate to Validated (to make the bullet lists consistent)?

Reply:
This isn't something that has been done, but something that is now performed when the code is run.